content/blog/2022-05-16-may-22-heartbeat.md

```diff
@@ -0,0 +1,395 @@
+
+He advocates for training your models using Python scripts, Git, and CI/CD to
+automatically shift your foucus to creating reusable, testable code, and to use
+tools like [Gradio](https://gradio.app/) and [Streamlit](https://streamlit.io/)
+to provide the interactivity of Jupyter notebooks. Sounds like a promising idea.
+💡
+
+https://media.giphy.com/media/qxtxlL4sFFle/giphy.gif
+
+## Beyond ML
+
+As noted above in our shiny new mission statement, our focus is to make tools
+for machine learning teams. It has however come to our attention that more and
+more users are using our tools for non-ML use cases.
+
+[**Dror Speiser**](https://drorspei.wordpress.com/about/) writes about a non-ML
+use case in
+[A New Recipe for Idempotent Cloud Deployments](https://drorspei.wordpress.com/2021/09/15/a-new-recipe-for-reproducible-cloud-deployments/)
+in which he provides a tutorial for doing just that with DVC.
+
+The benefits of the approach are:
+
+> 1. Changing one artifact’s code does not force rebuilding other artifacts,
+>    even if you’re building on a new VM every time.
+> 2. Changing only the deployment script won’t build any artifacts at all.
+> 3. You have an artifact repository that just works.
+> 4. Your Git history contains the hashes of all built artifacts.
+> 5. You can look up any artifact using its hash.
+
+We have opened up a #beyond-ml channel in our
+[Discord Server](https://dvc.org/chat). Do stop by and chat about alternate uses
+for our tools!
+
+## Upcoming Events
+
+- 📣 Our next in-person conference will be
+  [MLOps World](https://mlopsworld.com/) from June 7-10 in Toronto! We look
+  forward to seeing Community members there!
+- 📣 PyLadies Berlin is hosting **Doreen**, a data scientist working at
+  [Opinary](https://opinary.com/), who will be presenting "Reproducible Machine
+  Learning with DVC and Poetry" on May 17th.
+  [Join the event here.](https://www.meetup.com/PyLadies-Berlin/events/285313817/)
+- 📣 [**Nicolás Eiras**](https://www.linkedin.com/in/nicolas-eiris/) will be
+  presenting "Data Versioning: Towards Reproducibility in Machine Learning" at
+  [Embedded Vision Summit](https://embeddedvisionsummit.com/2022/session/data-versioning-towards-reproducibility-in-machine-learning/)
+  on May 18th in Santa Clara, California.
+- 📣 [Montreal PyData](https://www.meetup.com/PyData-MTL/) will host a
+  [Meetup](https://www.meetup.com/PyData-MTL/events/285894672/) on June 16th
+  with two presentations, "Introduction to Trustworthy Machine Learning for the
+  Enterprise" by [**Mohamed Leila**](https://www.linkedin.com/in/mohamedleila/),
+  ServiceNow and "ML in production in the video game industry: Ubisoft's use
+  case" by
+  [**Jean-Michel Daignan**](https://www.linkedin.com/in/jeanmicheldaignan/),
+  Ubisoft
+
+## Other Fun Stuff
+
+- [New Awesome list](https://github.com/gaocegege/awesome-open-source-mlops)
+- [New Udemy Course including DVC](https://www.udemy.com/course/dvc-and-git-for-data-science/)
```

Comment:
This link seems fine, Udemy is just refusing the link checker because it's either blocking bots or requiring logins.

Reply:
Thanks @rogermparent!!! 🙏